images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -363,6 +366,17 @@ _ocp_enabled_overrides:
         prometheus:
           url: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
 
+  presto:
+    spec:
+      config:
+        connectors:
+          prometheus:
+            enabled: true
+            config:
+              uri: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
+            auth:
```

Comment:
nitpick: let's create a newline between the `config` and `auth` sections.

Reply:
Looking around just in `main.yml` I don't see that convention in use throughout. Should we try to start having some standard for vertical whitespace in yaml? We could start by making that whole file conform. I'm up for that.

Best would be if we had some tool that could format and enforce whatever we want so all the yaml kinda starts looking more the same at least within our own project. Not just whitespace, but ordering of fields might be helpful too.

The Presto project does this for Java code via a Maven plugin that gets run in the verify phase of compile. For us it could be part of the make driven stuff I'd think.

Reply:
I was under the impression that we did have some sort of convention used throughout the default values file as we do in the MeteringConfig values.yaml. I think going forward we may need to clean up this ever-growing default values file.

Reply:
anyways.. I just added the one space for now. Done in 018461af